autopush/tests/test_log_check.py

```diff
@@ -1,18 +1,17 @@
 import json
 
 import twisted
-from cyclone.web import Application
-from twisted.trial import unittest
-from mock import Mock
-from moto import mock_dynamodb2
-from nose.tools import eq_
-
 from autopush.db import (
     create_rotating_message_table,
 )
-from autopush.log_check import LogCheckHandler
 from autopush.settings import AutopushSettings
+from autopush.web.log_check import LogCheckHandler
+from cyclone.web import Application
+from mock import Mock
+from moto import mock_dynamodb2
+from nose.tools import eq_
 from twisted.internet.defer import Deferred
+from twisted.trial import unittest
```

Comment:
I'm not sure how this violates PEP8. It's a group of non-local imports, ordered with a break before the local modules. It's a single entry, so no grouping is required.